docs/edgeql/lexical.rst

```diff
@@ -438,14 +438,14 @@ EdgeQL operators listed in order of precedence from lowest to highest:
     * - :eql:op:`*<MULT>`, :eql:op:`/<DIV>`,
         :eql:op:`//<FLOORDIV>`, :eql:op:`%<MOD>`
     * - :eql:op:`??<COALESCE>`
-    * - :eql:op:`DISTINCT`, unary :eql:op:`-<UMINUS>`
+    * - :eql:op:`DISTINCT`, :eql:op:`EXISTS`,
+	    :ref:`DETACHED <ref_eql_with_detached>`, unary :eql:op:`-<UMINUS>`
```

Comment:
Curios, why isn't `DETACHED` an `eql:op:` like everything else?  @vpetrovykh?

Reply:
Huh, it's documented as part of `WITH`?

Reply:
Yeah, comparing it to a top-level `WITH` block seemed like the simplest way to describe this operator, so it's ended up in that part of the documentation rather than with the rest of the operators. Basically, this operator messes with scopes and symbols, rather than actually just doing something to the set of values it is applied to.